### PR TITLE
Payment Element

### PR DESCRIPTION
**Pull request by MahoMuri**
🎨 style(PaymentElement.vue): format PaymentElement component code to improve code consistency and readability

### CODE REVIEW

File: .npmignore
Comment (jofftiquez) [inline .npmignore:1]: Please revert to old state

File: .gitignore
Comment (jofftiquez) [inline .gitignore:1]: Revert

File: .github/workflows/main.yml
Comment (jofftiquez) [inline .github/workflows/main.yml:1]: Do not delete yet

File: .github/workflows/codeql-analysis.yml
Comment (jofftiquez) [inline .github/workflows/codeql-analysis.yml:1]: Do not delete

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (jofftiquez) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Do not delete

File: .github/FUNDING.yml
Comment (jofftiquez) [inline .github/FUNDING.yml:1]: Do note delete

**Review by jofftiquez — APPROVED** [review]

**Review by jofftiquez — APPROVED** [review]

**Comment by jofftiquez** [comment]
No comments so far. Please provide recording of test app

File: src/stripe/LinkAuthenticationElement.vue [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
```
Comment (jofftiquez) [inline src/stripe/LinkAuthenticationElement.vue:2]: Omit extra div

File: src/stripe/LinkAuthenticationElement.vue [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <div id="link-authentication-mount-point"></div>
+  </div>
+</template>
+
+<script>
+import { loadStripe } from '@stripe/stripe-js';
+import { ref, onMounted } from 'vue';
+import {
```
Comment (jofftiquez) [inline src/stripe/LinkAuthenticationElement.vue:10]: Make one-line for now

File: src/stripe/LinkAuthenticationElement.vue [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  setup (props, { emit }) {
+    const stripe = ref(null);
+    const elements = ref(null);
+    const linkAuthElement = ref(null);
+
+    onMounted(async () => {
```
Comment (jofftiquez) [inline src/stripe/LinkAuthenticationElement.vue:47]: Move everything inside that's inside of `onMounted` to a new function called `init` and invoke it instead.

```js
onMounted(() => {
	init();
});

async function init () {
 // code here
}
```

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: src/stripe/LinkAuthenticationElement.vue (hunk 1 above)
Comment (MahoMuri) [reply src/stripe/LinkAuthenticationElement.vue:2]: Okay po

File: src/stripe/LinkAuthenticationElement.vue (hunk 2 above)
Comment (MahoMuri) [reply src/stripe/LinkAuthenticationElement.vue:10]: Okay po, done.

File: src/stripe/LinkAuthenticationElement.vue (hunk 3 above)
Comment (MahoMuri) [reply src/stripe/LinkAuthenticationElement.vue:47]: ganto po ba?

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: src/stripe/LinkAuthenticationElement.vue [hunk 4]
```diff
@@ -66,6 +64,10 @@ export default {
         LINK_AUTHENTICATION_ELEMENT_TYPE,
       );
       linkAuthElement.value.mount('#link-authentication-mount-point');
+    }
+
+    onMounted(async () => {
```
[the remark this answers is not in the document]
Comment (MahoMuri) [reply src/stripe/LinkAuthenticationElement.vue:69]: Ahhh I see, on it.

**Pull request merged**